src/prog_models/utils/calc_error.py

```diff
@@ -197,16 +170,16 @@ def MSE(m, times: List[float], inputs: List[dict], outputs: List[dict], **kwargs
             # For example, in LSTM models, the first few inputs will not 
             # produce an output until the model has received enough data
             # This is true for any window-based model
-            if any(np.isnan(z_obs.matrix)):
-                if counter < cutoffThreshold:
+            if any (np.isnan(z_obs.matrix)):
```

Comment:
```suggestion
            if any(np.isnan(z_obs.matrix)):
```
No space in function call